apps/prairielearn/public/stylesheets/colors.scss

```
@@ -2,16 +2,16 @@
 @import '../../../../node_modules/bootstrap/scss/mixins';
 
-// We reuse Bootstrap's `button-variant` mixin to generate appropriate hover
-// styles. Crucially, this will ensure that appropriate color contrast is
-// maintained.
-//
-// As with normal buttons (see `local.css`), we also include an outline with
-// `:focus-visible` to ensure that the focus state has sufficient contrast,
-// no matter the background color on which the button is situated.
+// Custom color classes used for badges, clickable badge-buttons, and other
+// colored UI elements throughout PrairieLearn.
 //
```

Stylelint (17.3.0)

[error] Unexpected empty comment (scss/comment-no-empty)

(scss/comment-no-empty)